.github/workflows/tests.yaml

```diff
@@ -20,8 +20,7 @@ on:
     workflow_dispatch:
 
 concurrency:
-    group:
```

Comment:
Why the random reformatting throughout this file?

Reply:
I assumed it was the restyle that did it. I removed all the style changes.